tt_eager/tt_dnn/op_library/sliding_window_op_infra/tt_py_max_pool.py

```diff
@@ -220,7 +221,7 @@ def max_pool_(activation):
 
             if self.deallocate_activation:
                 activation.deallocate()
-            output = ttl.tensor.max_pool2d_v2(
+            output = pool.max_pool2d_v2(
```

Comment:
`output_mem_config` should be changed to `memory_config` and should me a kw_only argument for consistency with the rest of ttnn ops.

Reply:
Changed